### DISCUSSION TITLE
Any plan to support serverless (with http) with openfaas?

### DISCUSSION THREAD

**Discussion by alexdonh** [TypedProject/ts-express-decorators#1664] [Q&A]
Hi @Romakita 

Do you have any plan to support `openfaas` or any hints of getting that supported? as you know i've been experimenting `Serverless.com` and also want to give `openfaas` a try.

Any help would be appreciated!

**Comment by Romakita** [comment]
Hello @alexdonh 

I think with the latest version of Ts.ED, it should works. I won't write tutorials or plugins for each Serverless provider!  But your feedback is welcome ;)

If you look the documentation here: https://www.openfaas.com/blog/serverless-nodejs/, the principle is the same as Vercel. So, you can start your app by using this example: https://github.com/tsedio/tsed-example-vercel

To begin, use the version commented in the code (`main.ts`): 

```

// WITH FULL http server
// async function bootstrap() {
//   const platform = await PlatformExpress.bootstrap(Server, {
//     httpsPort: false,
//     httpPort: process.env.PORT || 8000,
//     disableComponentsScan: true
//   });
//
//   await platform.listen();
//
//   return platform;
// }
//
// bootstrap();
```
Tell me you need more help ;)
See you :)
